distributed/protocol/tests/test_serialize.py

```diff
@@ -388,3 +389,27 @@ def _(x):
 
     header, frames = serialize([MyObj(), MyObj()])
     assert header["compression"] == [False, False]
+
+
+@pytest.mark.parametrize(
+    "test_pair",
```

Comment:
You can replace this with two terms separated by a comma like the following:

```suggestion
    "data,is_serializable",
```

Reply:
Neat - Thanks for the tip